pkg/agent/datamodel/types.go

```diff
@@ -1118,6 +1161,33 @@ func (w *WindowsProfile) GetWindowsDockerVersion() string {
 	return KubernetesWindowsDockerVersion
 }
 
+// GetWindowsDefaultRuntimeHandler get the default containerd runtime handler or return default value
+func (w *WindowsProfile) GetWindowsDefaultRuntimeHandler() string {
+	if w.WindowsRuntimes != nil && w.WindowsRuntimes.Default != "" {
+		return w.WindowsRuntimes.Default
+	}
+
+	return KubernetesDefaultWindowsRuntimeHandler
+}
```

Comment:
the same here, can we have RP pass in the final value?

Reply:
Sure, thanks, will do this in my next PR after RP has the default value.